alacritty/Cargo.toml

```diff
@@ -32,13 +32,13 @@ glutin = { version = "0.30.4", default-features = false, features = ["egl", "wgl
 home = "0.5.5"
 libc = "0.2"
 log = { version = "0.4", features = ["std", "serde"] }
-notify = "5.1.0"
+notify = "6.1.1"
```

Comment:
It hasn't fixed anything? Like partial writes, etc?

Reply:
Nothing particularly relevant as far as I can see:
https://github.com/notify-rs/notify/blob/HEAD/CHANGELOG.md#notify-600-2023-05-17